cpu/stm32f4/Makefile.features

```diff
@@ -1,4 +1,5 @@
 FEATURES_PROVIDED += periph_hwrng
+FEATURES_PROVIDED += puf_sram
```

Comment:
Why only stm32f4 and not all STMs ? (same question for samd21, why not all sam0?)

Reply:
@aabadie in order to use start-up patterns of SRAM memory as entropy source, the properties need to be evaluated. Yet, I only did this with a samr21-xpro, nucleo-f411re and re-mote. The plan is to add more boards consecutively.

Reply:
> The plan is to add more boards consecutively.

The problem with this approach is that f4 is not only nucleo-f411re but all boards with a stm32f4. Are they also evaluated ?
Maybe I'm wrong but I would move this in a more central place first, `cortexm_common/Makefile.features` sounds more appropriate, since it's a cortexm only feature.

Reply:
@aabadie I don't quite understand. 

>  f4 is not only nucleo-f411re but all boards with a stm32f4

You want to reduce the scope for that feature to nucleo-f411re?

> I would move this in a more central place first, cortexm_common/Makefile.features sounds more appropriate

You want to centralise and extend the feature to all Cortex-M platforms?

The assumption is, that the STM32F4 MCU has same SRAM properties on different boards. I just tried with the mentioned Nucleo as well as the STM32F4Disco board. Most likely, the other STM32FXYZ will behaves similar, but I did not yet evaluate this.

Currently this feature only works for Cortex-M platforms as you might have seen in the hook in *cortexm_common/vectors_cortexm.c*. But as mentioned, it is planned to extend this to other architectures.